apis/fluent-en_GB/extensions/jdk8/atrium-api-fluent-en_GB-jdk8-jvm/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/jdk8/fileAssertions.kt

```diff
@@ -0,0 +1,12 @@
+package ch.tutteli.atrium.api.fluent.en_GB.jdk8
+
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.domain.builders.ExpectImpl
+import java.io.File
+import java.nio.file.Path
+
+fun <T : File> Expect<T>.asPath(): Expect<Path> =
+    ExpectImpl.changeSubject.unreported(this) { it.toPath() }
+
+fun <T : File> Expect<T>.asPath(assertionCreator: Expect<Path>.() -> Unit): Expect<Path> =
```

Comment:
This should return `Expect<T>`, wrap the whole expression into an `apply` (see sequenceAssertions for an example)

Reply:
Fixed